test/src/go.mod

```
@@ -1,4 +1,4 @@
-module github.com/cloudposse/terraform-example-module
+module github.com/cloudposse/terraform-aws-ssm-patch-manager
 
 go 1.13
```

```suggestion
go 1.16
```

please empty this file and then run `go mod tidy` to update to the latest versions of the modules 

https://golang.org/ref/mod#go-mod-tidy